### PR TITLE
Refactor/remove functions 6

### PR DESCRIPTION
**Pull request by pranavtifr**

### CODE REVIEW

File: client/mapctrl_common.cpp [hunk 1]
```diff
@@ -66,24 +66,6 @@ struct city *city_workers_display = nullptr;
 static void clipboard_send_production_packet(struct city *pcity);
 static void define_tiles_within_rectangle(bool append);
 
-/**
-   Called when Right Mouse Button is depressed. Record the canvas
-   coordinates of the center of the tile, which may be unreal. This
```
Comment (lmoureaux) [inline client/mapctrl_common.cpp:71]: These two sentences were pure gold.

File: client/options.cpp [hunk 2]
```diff
@@ -113,13 +113,11 @@ struct client_options gui_options = {
     true,                           //.show_previous_turn_messages =
     false,                          //.concise_city_production =
     false,                          //.auto_turn_done =
-    true,                           //.meta_accelerators =
     true,                           //.ask_city_name =
     true,                           //.popup_new_cities =
     true,                           //.popup_actor_arrival =
     true,                           //.popup_attack_actions =
     true,                           //.popup_last_move_to_allied =
-    true,                           //.update_city_text_in_refresh_tile =
```
Comment (lmoureaux) [inline client/options.cpp:122]: You need to remove the corresponding initializers or everything will be shifted.

**Review by lmoureaux — CHANGES_REQUESTED** [review]

File: client/options.cpp (hunk 2 above)
Comment (pranavtifr) [reply client/options.cpp:122]: I did. Thats why i deleted these.

Comment (pranavtifr) [reply client/options.cpp:122]: Check line 93,99 in options.h

Comment (lmoureaux) [reply client/options.cpp:122]: Oooh they already had no
```cpp
    GEN_BOOL_OPTION(
        auto_turn_done, N_("End turn when done moving"),
        N_("Setting this option makes your turn end automatically "
           "when all your units are done moving."),
        COC_INTERFACE, false, nullptr),
```

**Review by lmoureaux — APPROVED** [review]

**Pull request merged**